.github/workflows/php_checks.yml

```diff
@@ -0,0 +1,41 @@
+name: PHPStan
+
+on:
+    - pull_request
+
+jobs:
+    ecs:
+        runs-on: ubuntu-latest
+
+        steps:
+            -
+                name: Checkout code
+                uses: actions/checkout@v2
+
+            -
+                name: Setup PHP
+                uses: shivammathur/setup-php@v1
+                with:
+                    php-version: 7.3
+                    coverage: none
+
+            -
+                name: Install
+                run:
+                    composer install --no-progress
+
+            -
+                name: Run
+                run:
+                    composer fix-cs
+
+            # this is so cool!
+            # see https://github.com/stefanzweifel/git-auto-commit-action
+            -
+                name: Commit changed files
+                uses: stefanzweifel/git-auto-commit-action@master
+                with:
+                    commit_message: Apply ECS changes
+                    branch: ${{ github.head_ref }}
+                env:
+                    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@stefanzweifel Hi, could I also you for review of this action?
I got stuck on it and I have no idea why it doesn't work :/
I basically copy pasted your example

Reply:
Thank you

Reply:
This issue usually related to a problem with git and the checked out branch. I've seen you're using checkout@v2. Haven't used that yet. Will give it a try in my own repo :)

Reply:
Ah, thanks :). I've disabled it for now and onyl run on `master`, as suggested in of issues https://github.com/stefanzweifel/git-auto-commit-action/issues/25#issuecomment-570803329

Reply:
I'm trying this, because Github doesn't allow me to create `GITHUB_TOKEN` in secrets:

```diff
-GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```